src/goose.rs

```diff
@@ -1527,19 +1528,20 @@ impl GooseUser {
             }
         };
 
-        // Send raw request object to parent if we're tracking statistics.
+        // Send a copy of the raw request object to the parent process if
+        // we're tracking statistics.
         if !self.config.no_stats {
-            self.send_to_parent(&raw_request)?;
+            self.send_to_parent(GooseMetric::Request(raw_request.clone()))?;
```

Comment:
nit - for potential future follow-up:

The need for a different variable name as `request` would shadow the other `request` shows that the processing here likely belongs into a helper function.

Reply:
Which are you suggesting to rename, `GooseMetric::Request`? Or `raw_request`? What would you rename it too? I'm not sure I'm understanding this comment.

Reply:
`request` - the other invocations of `send_to_parent` use:

```rust
self.send_to_parent(GooseMetric::Request(request.clone()))?;
```